src/components/molecules/PromptDialogBody.tsx

```diff
@@ -8,36 +8,54 @@ import {
   DialogButtonGroup,
   DialogButton,
 } from '../atoms/dialog/styled'
+import DialogPromptColorPicker from '../atoms/dialog/DialogPromptColorPicker'
 
 type PromptDialogProps = {
   data: PromptDialogOptions
   closeDialog: () => void
 }
 
 type PromptDialogState = {
-  value: string
+  value: PromptDialogValues
```

Comment:
PromptDialog should take only a `string` because it is supposed to behave like `window.prompt`. Please make anther Dialog component to deal with this.

Reply:
Change done!